client/components/nux-notice/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import Gridicon from 'gridicons';
+import { translate as __ } from 'i18n-calypso';
+
+const NuxNotice = ( { children, onDismiss } ) => {
```

Comment:
It's only used once and it's a small component, but it has substantial custom styles (see its associated `style.scss` file). It's about separation of concerns, I don't think those styles should live in the parent component.